### PR TITLE
fix: Update the important notice and github issue linking for this re…

### PR DESCRIPTION
**Pull request by tylerjusfly**
…pos pull request template

#### This pull request makes the following changes

* Fixes collabo-community/issue-tickets-ready-for-fixing#

#

#### General checklist

- [x] My pull request adheres to the [Collabo Community pull request guidelines](https://docs.collabocommunity.com/pull-request-guidelines).
- [x] I have read and understood the **Pull request guideline extension** section found at the bottom of this pull request description/body.
- [x] I have replaced the dummy text under the subheading **Testing checklist** below with the testing checklist from the issue ticket I'm working on.

#

#### Testing checklist

- [x] The important notice and github issue linking has been updated for the pull request template.
- [x] I certify that I ran my checklist

#

#### Pull request guideline extension

> [!NOTE]  
> Do not remove this note or section. This talks about extra things to take note of, in addition to the directions given in the Collabo Community pull request guidelines.

We appreciate the time taken to help tackle issue tickets within the Collabo Community. Please go ahead to submit your pull request even if it is not yet completed, and push to the pull request actively. This will allow us to know that you have not abandoned the issue ticket. However, you should only alert reviewers about your pull request as explained below, to avoid flooding their GitHub notifications unnecessarily.

"Ready for review" implies that things are working as expected, according to how it is specified in the issue ticket you are working on. Ensure that you test the code that you are working on, that it works as expected. Don't just request for review when your pull request is not yet done and working.

The only time you should be making people review something incomplete is: if you want to show them what's not working, so that they can send help or guidance. If this is the case, be explicit about this in our Discord community or in the comment of your pull request (after you have submitted the pull request).

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -20,7 +20,10 @@
 
 #### Testing checklist
 
-Replace this dummy text with the testing checklist from the issue this pull request fixes (use checkboxes similar to how it's done in the **General checklist** above).
+Note (to contributor taking on this task assignment): Add the testing checklist below (without this note) to your pull request when submitting your fix. This will guide (you and) the contributors who will review your fix to know what is important to check or test for.
+
+- [ ] The important notice and github issue linking has been updated for the pull request template.
+- [ ] I certify that I ran my checklist
```
Comment (Ifycode) [inline .github/pull_request_template.md:26]: I'm trying to understand why you are changing testing checklist too @tylerjusfly 
The issue didn't mention to do this.

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (Ifycode) [reply .github/pull_request_template.md:26]: Also why is your branch name not using our new naming convention? @tylerjusfly 
Check that Discord thread.

Comment (Ifycode) [reply .github/pull_request_template.md:26]: Also assign yourself to the ticket, that direction you will also find in the thread @tylerjusfly

Comment (tylerjusfly) [reply .github/pull_request_template.md:26]: > I'm trying to understand why you are changing testing checklist too @tylerjusfly The issue didn't mention to do this.

oh really ? okay i will revert that and push

**Comment by tylerjusfly** [comment]
Created a new PR with the correct branch name

Link to new PR https://github.com/collabo-community/issue-tickets-ready-for-fixing/pull/11

File: .github/pull_request_template.md (hunk 1 above)
Comment (tylerjusfly) [reply .github/pull_request_template.md:26]: Sorry for the oversight.

**Pull request closed without merging**